packages/web/src/package/observers/invisible/secret.ts

```diff
@@ -56,17 +58,14 @@ export function decodeFromText(text: string) {
   const invisibleMessages = text
     .match(INVISIBLE_REGEX)
     ?.filter((m) => m.length > 8);
-  return invisibleMessages?.map(decodeMessage) || [];
+  const result = [];
+  invisibleMessages?.map(decodeMessage).forEach((message) => {
+    result.push(message);
+  });
+
+  return result;
```

Comment:
Why this change? The previous version seems to do the same thing essentially. (Am I missing something?)